### ISSUE TITLE
Error when merged.mp4 already exists with --merge option

### ISSUE THREAD

**Issue by MelvinKool** [MelvinKool/TUEDownloader#6]
If e.g. a channel is downloaded and some directories already contain merged.mp4, ffmpeg will prompt you whether to overwrite the file.
If you select no, an exception will be raised.
Maybe we could add a check to see if merged.mp4 already exists.

```
File './2 Mathematics - 2MMC10 2019-2020/2MMC10 - Lecture 14 b/merged.mp4' already exists. Overwrite ? [y/N] n
Not overwriting - exiting
Traceback (most recent call last):
  File "/usr/local/bin/tuedownloader", line 11, in <module>
    load_entry_point('TUEDownloader==0.1', 'console_scripts', 'tuedownloader')()
  File "/usr/local/lib/python3.8/site-packages/TUEDownloader-0.1-py3.8.egg/tuedownloader/download.py", line 434, in main
  File "/usr/local/lib/python3.8/site-packages/TUEDownloader-0.1-py3.8.egg/tuedownloader/download.py", line 394, in download_channel
  File "/usr/local/lib/python3.8/site-packages/TUEDownloader-0.1-py3.8.egg/tuedownloader/download.py", line 264, in download_video_showcase
  File "/usr/local/lib/python3.8/site-packages/TUEDownloader-0.1-py3.8.egg/tuedownloader/editor.py", line 84, in diagonal
  File "/usr/local/lib/python3.8/site-packages/ffmpeg_python-0.2.0-py3.8.egg/ffmpeg/_run.py", line 325, in run
    raise Error('ffmpeg', out, err)
ffmpeg._run.Error: ffmpeg error (see stderr output for detail)
```

**Comment by MelvinKool** [comment]
Fixed in cfa17e66f22ae7958726e53233d8f855351744cf, needs to be tested.

**Comment by MelvinKool** [comment]
Tested and it works, merged in master, closing issue.
